tests/unit/test_x86.c

```diff
@@ -729,6 +729,56 @@ static void test_x86_16(void **state)
 
 /******************************************************************************/
 
+
+/******************************************************************************/
+
+// callback for SYSCALL instruction (X86).
```

Comment:
No, sorry this is a copy and paste mistake.

On 02.02.2016 16:08, Nguyen Anh Quynh wrote:
> In tests/unit/test_x86.c
> <https://github.com/unicorn-engine/unicorn/pull/413#discussion_r51580037>:
>
> > @@ -729,6 +729,56 @@ static void test_x86_16(void **state)
> >  
> >  /******************************************************************************/
> >  
> > +
> > +/******************************************************************************/
> > +
> > +// callback for SYSCALL instruction (X86).
>
> is this a correct comment? i dont see anything about SYSCALL here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/unicorn-engine/unicorn/pull/413/files#r51580037>.
>